docker/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata
+# Add sudo
+RUN apt-get -y install sudo
+RUN apt-get install -y --no-install-recommends software-properties-common
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt update
+RUN apt install -y --no-install-recommends python3.6
```

Comment:
```
RUN apt update
RUN apt install -y --no-install-recommends python3.6
``` 
doesn't seem to work anymore
`RUN apt-get install -y python3.6` + using `nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04` as base image fixed that for me